api/proto/config/devmodel.proto

```diff
@@ -68,11 +68,8 @@ message SystemAdapter {
         string networkUUID = 4;
 
         // if its static network we need ip address
+        // If this is specified, networkUUID must be ""
```

Comment:
I thought I commented on this already, but I think the comment is reversed. networkUUID must be set to a valid UUID if the IP address is set.

Reply:
In general that is not required, for instance if we ever support pt-pt interfaces.
So we don't need to add that check on the device.

Reply:
But note that my comment is to just change from 'must be ""'  to 'must be set'

Reply:
Changed it as:

        // addr - if its static network we need ip address
        // If this is specified, networkUUID must also be specified. addr
        // is expected to be in sync with the network object ( same subnet etc ).